Translate the project to PT-BR

airtonzanon opened the pull request:
Now we have the project in Portuguese (Brazil) then we can add
Portuguese articles here to help people to start do public
speaking.

This commit start the translations that we discussed before, so we just resolve #21

airtonzanon committed:
Translate the project to PT-BR

Now we have the project in Portuguese (Brazil) then we can add
Portuguese articles here to help people to start do public
speaking.

vmbrasseur commented:
Re: #27, we're going to include the Portuguese articles in the main list. We need the English-only speakers to become more aware of the fact that the rest of the world exists, and this is a subtle way to do that. ;-)

vmbrasseur commented on translations/pt-br/CONTRIBUTING.md:
```diff
@@ -0,0 +1,40 @@
+
+Olá! Obrigado por querer contribuir com este projeto! Nós somos gratos aos nossos contribuidores.
+
+Todos os contribuidores concordam com o [Código de Conduta](./CODEOFCONDUCT.md).
```
Probably should link to the PT-BR version here.

vmbrasseur replied:
Ah, wait. I see now that it does since you kept the same filenames. Excellent. 😄

airtonzanon replied:
Yeah, I did it to follow a standard.

vmbrasseur commented on translations/pt-br/README.md:
```diff
@@ -0,0 +1,175 @@
+Artigos que podem ajudar pessoas a falar em público
```
Suggestion: Put a header here. Then it'll automatically form an anchor and I'll update the main README with a link directly to this section for PT-BR speakers.

vmbrasseur commented on translations/pt-br/README.md:
```diff
@@ -0,0 +1,175 @@
+
+Dê uma olhada abaixo ou clique aqui para ir direto na sessão:
+
+* [Geral](#geral)
```
Each of these should link back to the main README anchors instead of this doc. For now, we're going to include all of the resources there no matter which language they're in.

vmbrasseur commented on translations/pt-br/README.md:
```diff
@@ -0,0 +1,175 @@
+* [Apresentando a palestra](#apresentando)
+* [Outros](#outros)
+
+### Geral
```
These links are going to change far too frequently for them to be duplicated here. We'd forget to update the non-English document, making those non-English things second class citizens. We can't have that. Instead, probably best to remove all of the resources from here and link to the appropriate section in the main README instead.

vmbrasseur commented:
So the general idea (for now) is:

* All resources (no matter what language) in main README.md
* All section translations in the very smart structure you've set forth here
* I'll handle linking the main README section to the appropriate sections of the translation

If we have the resources split across multiple files, not only does it mean people don't see the resources available in all languages (many in technology are multi-lingual), it also causes a potential point of failure as people forget to add new resources to ALL of the files. I don't want to risk that or risk any non-English file becoming a second-class citizen here.

airtonzanon commented:
I really liked that idea to center all of the articles on the main, and just translate the English lines.

airtonzanon committed:
Link translation to main README

Now the translation will link the resources to main README so it
will prevent many of failure, as people forget to add new resources
to all tranlations.

airtonzanon commented:
I just updated the README linking it to the main README

vmbrasseur commented:
@airtonzanon This is **AMAZING**. ❤️
